SignalMessaging/attachments/OWSAudioAttachmentPlayer.m

```diff
@@ -48,7 +58,8 @@ - (void)dealloc
 {
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 
-    [DeviceSleepManager.sharedInstance removeBlockWithBlockObject:self];
+    // FIXME SHARINGEXTENSION
+    //  [DeviceSleepManager.sharedInstance removeBlockWithBlockObject:self];
```

Comment:
Add methods to AppContext?

Reply:
Right, sorry I meant to call this one out to make sure we're on the same page.

For the sharing extension context should this just be a no-op?

Reply:
Yup

Reply:
done!